msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md

```diff
@@ -91,9 +92,11 @@ Your service must respond with the results matching the user query. The response
 |`composeExtension.suggestedActions`|Suggested actions. Used for responses of type `auth` or `config`. |
 |`composeExtension.text`|Message to display. Used for responses of type `message`. |
 
-### Configuration response
+### `config` response type
 
-Configuration response is the data returned by the server or application to configure and enable the message extension within the messaging platform. The following code is an example for message extension configuration:
+The configuration response is the data returned by the server or application to configure and enable the message extension within the messaging platform. When a user configures the message extension for the first time, a config response is used to prompt the user to set up the message extension and provide any necessary configuration.
+
+The following code is an example of message extension configuration:
```

Comment:
Text says below code but the content below is a JSON. As I missing something here? Should this be a code snippet instead?